bot_modules/search_commands/providers.js

```diff
@@ -0,0 +1,107 @@
+const Discord = require('discord.js')
+const axios = require('axios')
+
+/**
+ * Handle the provider command
+ *
+ * @param {Discord.Message} msg The original message send by the user
+ * @param {Array} args The list of arguments passed to the command
+ */
+exports.handleProviderCommand = async (msg, args) => {
+  let embed = new Discord.MessageEmbed()
+
+  // Check to make sure we have a provider
+  if (args.length <= 1) {
+    embed.setTitle('Invalid usage')
+    embed.setDescription('Missing provider to check. `!provider <provider>`')
+    embed.setColor(0xff0000)
+    msg.channel.send(embed)
+    return
+  }
+
+  // Remove the first argument (the command)
+  args.splice(0, 1)
+
+  // Join the other arguments together and collect the providers
+  const query = args.join(' ')
+  const providers = await getProviders()
+  let foundProvider = false
+
+  providers.forEach((provider) => {
+    if (provider.name.toLowerCase().startsWith(query.toLowerCase())) {
```

Comment:
It would be nice if it accepted slight misspellings

Reply:
Also is the double space thing fixed? It doesn't look like it is, maybe do `query.trim()`?